alfresco-search/src/main/resources/solr/instance/templates/noRerank/conf/lang/synonyms_en.txt

```diff
@@ -0,0 +1,41 @@
+########### ENGLISH SYNONYMS
+# This file lists several scenarios where synonyms could be applied for improving the
+# search effectiveness.
+# Important: the current Alfresco configuration lowecases the input stream before the Synonyms management plays its role;
+# that's the reason why in this file all terms, concepts, are in lowercase.
```

Comment:
This doesn't seem to be true

Reply:
Nice shot! This was related with the first version of the file

Reply:
ok @tpage-alfresco I removed that description. 
Ideally, the synonyms management will be exposed through remote API (so it will be available from the admin console) and in such case it doesn't make sense to add a constraint like that to an admin or an end user. As an admin / user I want to define my synonyms with a proper casing.